website/templates/project/settings.mako

```diff
@@ -68,7 +68,27 @@
                   <span data-bind="css: messageClass, html: message"></span>
                   <span data-bind="if: disabled" class="help-block">
                     A top-level project's category cannot be changed
-                  </span>
+                  </span>               
+                  </p>
+                   
+                    <label for="title">Title</label>
+                    <input class="form-control" type="text" maxlength="200" data-bind="value: title, 
+                                                                            event: { keyup: dirtyTitleDescription }">
+                    <span id="title-input-message"></span>        
+                    <br>
```

Comment:
`<br>` shouldn't be used for layout. Wrapping with a `div.form-group` is probably what you want.

See Bootstrap's documentation for forms: http://getbootstrap.com/css/#forms-example